docs/content/contributing/terraform/terraform-contribution-flow/owner-contribution-flow.md

```diff
@@ -31,25 +35,88 @@ Make sure module authors/contributors tested their module in their environment b
+| [RMFR7](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-rmfr7---category-outputs---minimum-required-outputs) | Category: Outputs - Minimum Required Outputs |
+
+<br>
+
+---
+
+<br>
+
+### 2. GitHub Teams and repository creation and configuration
 
 Familiarise yourself with the AVM Resource Module Naming in the [module index csv's](https://github.com/Azure/Azure-Verified-Modules/tree/main/docs/static/module-indexes).
 
 - Example: `terraform-<provider>-avm-res-<rp>-<ARM resource type>`
 
-1. Create GitHub teams:
+{{< hint type=important >}}
+
+Make sure you have access to the Azure organisation see [GitHub Account Link and Access](https://azure.github.io/Azure-Verified-Modules/contributing/terraform/prerequisites/#github-account-link-and-access).
+
+{{< /hint >}}
+
+1. Create the module repostory using [terraform-azuremrm-avm-template](https://github.com/Azure/terraform-azurerm-avm-template) in the `Azure` organisation with the following [details (internal only)](https://dev.azure.com/CSUSolEng/Azure%20Verified%20Modules/_wiki/wikis/AVM%20Internal%20Wiki/333/-TF-Create-repository-in-Github-Azure-org-and-conduct-business-review). You will then have to complete the configuration of your repository and start an [internal business review](https://dev.azure.com/CSUSolEng/Azure%20Verified%20Modules/_wiki/wikis/AVM%20Internal%20Wiki/333/-TF-Create-repository-in-Github-Azure-org-and-conduct-business-review?anchor=conduct-initial-repo-configuration-and-trigger-business-review).
+
+2. Create GitHub teams as outlined in [SNFR20](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only) and add respective parent teams:
+
+Segments:
+
+- `avm-res-<RP>-<modulename>-module-owners-tf`
+- `avm-res-<RP>-<modulename>-module-contributors-tf`
+
+Examples:
 
-- `@Azure/avm-res-<RP>-<modulename>-module-owners-tf`
-- `@Azure/avm-res-<RP>-<modulename>-module-contributors-tf`
+<!-- TODO: Clarify parent teams with the team -->
 
-2. Create the module repostory using [terraform-azuremrm-avm-template](https://github.com/Azure/terraform-azurerm-avm-template) in the `Azure` organisation with the following [details (internal only)](https://dev.azure.com/CSUSolEng/Azure%20Verified%20Modules/_wiki/wikis/AVM%20Internal%20Wiki/333/-TF-Create-repository-in-Github-Azure-org-and-conduct-business-review). You will then have to complete the configuration of your repository and start an [internal business review](https://dev.azure.com/CSUSolEng/Azure%20Verified%20Modules/_wiki/wikis/AVM%20Internal%20Wiki/333/-TF-Create-repository-in-Github-Azure-org-and-conduct-business-review?anchor=conduct-initial-repo-configuration-and-trigger-business-review).
+- `avm-res-compute-virtualmachine-module-owners-tf` and added `avm-technical-reviewers-tf` as parent.
+- `avm-res-compute-virtualmachine-module-contributors-tf` and added `avm-technical-reviewers-tf` as parent.
```

Comment:
@matt-FFFFFF and @mbilalamjad this needs to be decided, if we go with parent teams we need to decide on the names. Parent teams are required for the teams permissions inheritance housekeeping.

Reply:
Why do we want them parented?

Reply:
Removed.